src/components/StatsScoringDialog.vue

```diff
@@ -41,26 +36,55 @@
         </p>
         <v-list>
           <v-list-item>
-            <v-icon :color="theme.firstPlace" class="mr-2" icon="mdi-medal" />
+            <v-chip
+              variant="elevated"
+              class="mr-2 mb-1"
+              :color="this.$vuetify.theme.themes.cuttleTheme.colors.firstPlace"
```

Comment:
For brevity's sake, let's define a computed property called `theme` that returns `theme()` that returns `this.$vuetify.theme.themes.cuttleTheme.colors` so that each of these colors can just be `theme.firstPlace`, `theme.secondPlace`, etc

Reply:
Ha! I see we have a `theme` comptuted property already but it's out of date. The above snippet is what it should return: 
```
theme() {
  return this.$vuetify.theme.themes.cuttleTheme.colors;
}
```

Reply:
Also there is a bit of a gotcha in Vue that the html/template code _doesn't_ need **this**, but the javascript _does._ So even without the computed property these colors should look like

```
:color="$vuetify.theme.themes.cuttleTheme.colors.firstPlace" // notice I removed 'this'
```
but this will be cleaner if we do fix the `theme` computed prop as written above and then this will just be

```
:color="theme.firstPlace"
```
which is easier to read and less repetitive given that we use the same expression 4x here

Reply:
Definitely reads much better this way, good call!